code/game/turfs/floors/natural/_natural.dm

```diff
@@ -37,24 +37,35 @@
 		color = base_color
 	else
 		color = null
-	if(material)
-		set_turf_materials(material, skip_update = no_update_icon)
 
 	if(possible_states > 0)
 		icon_state = "[rand(0, possible_states)]"
+
+	// TEMP: set these so putting tiles over natural turfs doesn't make green sand.
```

Comment:
what's this about? how is this going to be replaced/superseded later? not a big fan of temp code without a TODO explaining what we're keeping it around until